src/commands/run-task.yml

```diff
@@ -113,6 +113,11 @@ parameters:
     description: AWS profile name to be configured.
     type: string
     default: ''
+  run-task-output:
+    description: |
+          Specifies a local json file to save the output logs from the aws ecs run-task command. This will enable users to extract information like task-arn's, task-id's and more from the output.
```

Comment:
> task-arn's

Don't believe apostrophe should be here.

>This will enable users to extract information like task-arn's 

Not a fan of this phrasing. The person reading this will be the user.

Suggestion:

> Specifies a local json file to save the output logs from the aws ecs run-task command. Use tools like JQ to read and parse this information such as "task-arns" and "task-ids"

Reply:
changed the description